app/controllers/collection_participants_controller.rb

```diff
@@ -109,7 +109,14 @@ def add
     end
     
     if @participants_invited.empty? && @participants_added.empty?
-      flash[:error] = ts("We couldn't find anyone new by that name to add.")
+      if pseud_results[:banned_pseuds].present?
+        flash[:error] =
+            ts("%{name} is currently banned and cannot participate in Challenges.",
```

Comment:
"Challenges" probably doesn't need to be capitalised here. I don't think we do that elsewhere.